packages/blocks/src/surface-block/surface-model.ts

```diff
@@ -4,7 +4,7 @@ import { literal } from 'lit/static-html.js';
 export const SurfaceBlockSchema = defineBlockSchema({
   flavour: 'affine:surface',
   metadata: {
-    version: 3,
+    version: 4,
```

Comment:
Is it possible that we make this non-breaking and just set the `roughness` field a default value if it not exists?

Reply:
Do you mean to make this field optional?

Reply:
I think it can still be a required field, but maybe we can assign a default value to it, just in case it's missing (in the case we load content from old binaries) 🤔

Reply:
I can add a default value during rendering, but I believe this should only be used as a fallback strategy for migration failures.